Projects/Server/Gumps/New/Components/Interpolation/GumpInterpolatedStringHandler.Formatters.cs

```diff
@@ -0,0 +1,75 @@
+/*************************************************************************
+ * ModernUO                                                              *
+ * Copyright 2019-2023 - ModernUO Development Team                       *
+ * Email: hi@modernuo.com                                                *
+ * File: GumpInterpolatedStringHandler.Formatters.cs                     *
+ *                                                                       *
+ * This program is free software: you can redistribute it and/or modify  *
+ * it under the terms of the GNU General Public License as published by  *
+ * the Free Software Foundation, either version 3 of the License, or     *
+ * (at your option) any later version.                                   *
+ *                                                                       *
+ * You should have received a copy of the GNU General Public License     *
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>. *
+ *************************************************************************/
+
+namespace Server.Gumps.Components.Interpolation
+{
+    public static class GumpInterpolatedStringHandler
```

Comment:
We have a static class and ref struct with the same name. I am not sure how this even compiles? LOL - Should we distinguish them? Do we even need this top level class?

Reply:
Yep, this compiles because the ref struct is a generic one, so the type doesn't collide.
I renamed the static class and moved out the related interface